src/main/java/beam/analysis/summary/VehicleTravelTimeAnalysis.java

```diff
@@ -44,8 +46,9 @@
     private Map<String, List<Double>> personIdDelays = new HashMap<>();
     private Map<String, List<String>> personsByVehicleIds = new HashMap<>();
 
-    public VehicleTravelTimeAnalysis(Scenario scenario, scala.collection.Set<Id<BeamVehicleType>> vehicleTypes) {
+    public VehicleTravelTimeAnalysis(Scenario scenario, NetworkHelper networkHelper, scala.collection.Set<Id<BeamVehicleType>> vehicleTypes) {
```

Comment:
Why not follow unwritten standards of adding new params to the end of the list?

Reply:
Just feel a bit weird to pass `NetworkHelper networkHelper` after `Set<Id<BeamVehicleType>> vehicleTypes` (in this case prefer to pass single element at first and then collections). But I'm open to fix it.

Reply:
I'll take it. I was more curious than anything else.